internal/constants/version/version.go

```diff
@@ -88,20 +91,29 @@ func (v *Incrementation) Type() (string, error) {
 	return Zeroed, nil
 }
 
-func fetchLatestVersionString(branch string) (string, error) {
-	// linux-amd64.json is our single source of truth for the latest version number
-	stateURL := "https://s3.ca-central-1.amazonaws.com/cli-update/update/state/%s/linux-amd64.json"
-	resp, err := http.Get(fmt.Sprintf(stateURL, branch))
+func latestVersionString(branch string) (string, error) {
+	type versionJSON struct {
+		Version string
+	}
+	var v versionJSON
+
+	rootPath, err := environment.GetRootPath()
 	if err != nil {
 		return "", err
 	}
-	defer resp.Body.Close()
 
-	type versionJSON struct {
-		Version string
+	versionFilePath := filepath.Join(rootPath, "build", "version.json")
```

Comment:
This should probably be `build/update/version.json`, see http://github.com/ActiveState/cli/blob/c26bb7df7cfda729e20290a02f5177c24673d613/activestate.yaml#L70-L70